app/controllers/users_controller.rb

```diff
@@ -1,12 +1,15 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  before_action :logged_in_user, except: %i(new create show)
+  before_action :load_user, except: %i(index new create)
+  before_action :correct_user, only: %i(edit update)
+  before_action :check_admin, only: :destroy
 
-    flash[:danger] = t ".not_found"
-    redirect_to signup_path
+  def index
+    @users = User.paginate(page: params[:page], :per_page => Settings.per_page_10)
```

Comment:
Dùng như này nếu cần set mặc định
```suggestion
    @users = User.paginate(page: params[:page], per_page: Settings.per_page_10)
```